cql3/Cql.g

```diff
@@ -1014,6 +1014,7 @@ alterTableStatement returns [std::unique_ptr<alter_table_statement> expr]
 
 cfisStatic returns [bool isStaticColumn]
     @init{
+        $isStaticColumn = false;
```

Comment:
This is sad, it should not have been needed because the only rule for cfisStatic sets `$isStaticColumn`. So this smells like either an antlr bug (it may return from this rule without isStaticColumn being set) or a compiler bug - did you try to take a look which it is (and when you find out, document it it in the commit message)? Can you please look at line CqlParser.cpp:26645 reported by the error message, and see what it is trying to do there?

Anyway, if we are adding this extra initialization to appease the compiler, do we really need both the $isStaticColumn and the additional isStatic variable here? If you already set $isStaticColumn to false in init, maybe you don't need the second variable isStatic at all, because it's purpose was exactly the same -  to default to false?